js/application.js

```diff
@@ -7,132 +7,172 @@ var players = [];
 // ADDING PLAYERS
 
 var addNewPlayer = function(){
-	
-	$('#add_button').click(function(){
-		var newPlayerName = $('#input_new_player_name').val();
-		var newPlayer = '<li>' + newPlayerName + '</li>';
-		
-		if (newPlayerName == ""){
-			alert('Please input a name first');
-		}else{
-			$(newPlayer).appendTo("#list_of_players").hide().fadeIn(500);
-			players.push({"namePlayer": newPlayerName,"score": 0});
-			$('#input_new_player_name').val("");
-			console.log(players);
-		};
-
-	});
+    
+  $('#add_button').click(function(){
+    var newPlayerName = $('#input_new_player_name').val();
+    var newPlayer = '<li>' + newPlayerName + '</li>';
+    
+    if (newPlayerName == ""){
```

Comment:
You can prevent this from HTML itself. Read https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/Data_form_validation